dali/pipeline/operators/transpose/transpose.cu

```diff
@@ -111,10 +111,14 @@ void cuTTKernelBatched(const TensorList<GPUBackend>& input,
 }
 }  // namespace kernel
 
-template <>
-Transpose<GPUBackend>::~Transpose() {
+template<>
+Transpose<GPUBackend>::~Transpose() noexcept {
```

Comment:
I don't know if this `noexcept` is needed here as it no longer throws.

Reply:
Since throwing out of destructor is evil, I'd pass it. Theoretically dtors are implicitly `noexcept`, but there are some cases, that they aren't, so I'd pass it still... Works for me both ways, but I'd leave it.